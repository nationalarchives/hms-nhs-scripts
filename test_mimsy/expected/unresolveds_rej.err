Unresolved "admission number" at line 3 of input/unresolveds_rej.csv
Exception while parsing row 3 of input/unresolveds_rej.csv

