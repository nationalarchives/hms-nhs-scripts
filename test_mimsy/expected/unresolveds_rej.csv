DSH/1/91,admission number, 7175; date of entry, 05-05-1827; name, Edward Lawrence; quality, Able Seaman; age, 25; place of birth, Bemester; port sailed out of, ; years at sea, 00: 07; last services, Malta; under what circumstances admitted (or nature of complaint), Ulcered Leg; date of discharge, 04-06-1827; how disposed of, Request Cured; number of days victualled, 31; 